apps/coscrad-frontend/src/utils/generic-components/presenters/test-id-div.tsx

```diff
@@ -0,0 +1,9 @@
+interface TestIdDivProps {
+    testId: string;
+}
+
+export const TestIdDiv = ({ testId }: TestIdDivProps): JSX.Element => (
```

Comment:
Not a big fan of this, for reasons mentioned above.

Reply:
One argument in favour of this is that if you're typing it out you could forget the height or the `&nbsp;`.  I agree that it's a bit awkward, but I also don't like the pattern of using a containing div around the contents of a component just for a testid.  It's weird that the testing library doesn't support querying for a class name.  That would work with MUI.